guides/common/modules/proc_upgrading-project-from-el7-to-el8.adoc

```diff
@@ -1,15 +1,47 @@
-[id="Upgrading-Project-from-EL7-to-EL8_{context}"]
+[id="Upgrading-Project-from-Enterprise-Linux-7-to-Enterprise-Linux-8_{context}"]
+ifdef::foreman-el,katello[]
 = Upgrading {Project} from Enterprise Linux 7 to Enterprise Linux 8
+endif::[]
+ifdef::satellite[]
+= Upgrading {Project} from {RHEL} 7 to {RHEL} 8
+endif::[]
 
+ifdef::foreman-el,katello[]
 Use this procedure to upgrade your {Project} installation from Enterprise Linux 7 to Enterprise Linux 8.
 LEAPP is the supported tool for Enterprise Linux 7 to Enterprise Linux 8 upgrades.
+endif::[]
+
+ifdef::satellite[]
+Use this procedure to upgrade your {Project} installation from {RHEL} 7 to {RHEL} 8.
+endif::[]
 
+ifdef::foreman-el,katello[]
 .Prerequisites
 * {Project} {ProjectVersion} running on Enterprise Linux 7.
 * Access to available repositories or a local mirror of repositories.
-* No outstanding updates.
 * {Project} installations running on CentOS 7 can be upgraded to CentOS Stream 8 or a {RHEL} rebuild.
 * {Project} installations running on {RHEL} 7 can be upgraded to {RHEL} 8.
+endif::[]
+
+ifdef::satellite[]
+.Prerequisites
+* {Project} {ProjectVersion} running on {RHEL} 7.
+* Access to available repositories or a local mirror of repositories.
+* {Project} installations running on {RHEL} 7 can be upgraded to {RHEL} 8.
```

Comment:
Given RHEL7→RHEL8 is the only path available for Satellite, we could drop this sentence (it doesn't hurt, but it gives also zero new knowledge to the reader).